### PR TITLE
add bookmark conventions and improve profile

### PR DESCRIPTION
**Pull request by soltanireza65**
for the profile, I've used inrupt's sample app as a reference 
and for the bookmarks, I've looked a look at `https://github.com/mark-book/markbook`

### CODE REVIEW

File: bookmark.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Notepad
```
Comment (michielbdejong) [inline bookmark.md:1]: what does bookmark have to do with bookmarks? copy-paste error maybe?

File: bookmark.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+_NB: this currently mainly describes how Solid OS stores data on a Solid Pod._
```
Comment (michielbdejong) [inline bookmark.md:2]: what about mark-book, then?

**Review by michielbdejong — CHANGES_REQUESTED** [review]

File: bookmark.md (hunk 1 above)
Comment (soltanireza65) [reply bookmark.md]: fixed

File: bookmark.md (hunk 2 above)
Comment (soltanireza65) [reply bookmark.md]: fixed

**Pull request merged**